### ISSUE TITLE
Version: Purpur 1.21

### ISSUE THREAD

**Issue by PixelNube** [kangarko/ChatControl#2805]
### Are you using MySQL?

No

### Are you using a proxy?

No

### "/version ChatControlRed" - plugin version

10.27.7

### Optional: "/chc debug" file

I sent it to support@mineacademy.org, title is has "PixelNube Github Debug Files" and a link that redirects to this complaint

### Steps to reproduce

![image](https://github.com/user-attachments/assets/ec7df3e2-0f06-4ad8-8165-78765fe357c2)
![image](https://github.com/user-attachments/assets/f61612ca-3258-4693-b1f9-9da4ae1717e0)
![image](https://github.com/user-attachments/assets/4d08bc53-b411-44dc-b44d-7a2a5fbcd868)

Hello, any idea why this placeholder isn't working?
When I papi parse it, it works

**Comment by kangarko** [comment]
Please use the {} syntax now %%. Any other issues should be fixed in today's version.

**Comment by PixelNube** [comment]
That would make it so it doesn't work with other plugins tho as only CHC uses {} for placeholders and this was set as the "player/default" group prefix.

SOLUTION: 
Turns out I had to give the "Receiver_Permission" and "Sender_Permission" permissions to the default luckperms group, "chatcontrol.group.cosychat" and "chatcontrol.hasprefix" in this case. This fixed it.

![image](https://github.com/user-attachments/assets/919d0d22-d18b-41c2-8cdf-3cd0c23db3f6)
